include/crm/common/acl_internal.h

```diff
@@ -10,6 +10,8 @@
 #ifndef CRM_COMMON_ACL_INTERNAL__H
 #define CRM_COMMON_ACL_INTERNAL__H
 
+#include <string.h>      // strcmp()
```

Comment:
We've been lucky, in both files that call `pcmk__is_privileged` get `string.h` through a separate include.